optimizely/helpers/condition.py

```diff
@@ -233,12 +238,74 @@ def substring_evaluator(self, index):
 
         return condition_value in user_value
 
+    def semver_equal_evaluator(self, index):
```

Comment:
Please add doc string for all the new methods. See other method docs for reference.